core/src/types/operator/operator.rs

```diff
@@ -29,7 +29,9 @@ use tokio::io::ReadBuf;
 
 use super::BlockingOperator;
 use crate::ops::*;
+use crate::raw::oio::Append;
 use crate::raw::*;
+use crate::types::appender::Appender;
```

Comment:
`Appender` should be exposed as `opendal::Appender` which covered by `use crate::*`.

Reply:
Added automatically by IDE, removed.